src/oscar/apps/catalogue/expressions.py

```diff
@@ -0,0 +1,34 @@
+from django.db.models.expressions import Subquery
+
+EXPAND_UPWARDS_CATEGORY_QUERY = """
```

Comment:
Dare I even ask whether this works with MySQL? :stuck_out_tongue: I know we don't officially support it - just be prepared for some complaints anyway if it doesnt.

Reply:
There is no reason this shouldn't work in MySQL, however I haven't tested it because I don;t use it and we don't support it.